test/PublishPSResource.Tests.ps1

```diff
@@ -11,14 +11,14 @@ Describe "Test Publish-PSResource" {
         $tmpRepoPath = Join-Path -Path $TestDrive -ChildPath "tmpRepoPath"
         New-Item $tmpRepoPath -Itemtype directory -Force
         $testRepository = "testRepository"
-        Register-PSResourceRepository -Name $testRepository -URL $tmpRepoPath -Priority 1 -ErrorAction SilentlyContinue        
-        $script:repositoryPath = (get-psresourcerepository "testRepository").Url.AbsolutePath 
+        Register-PSResourceRepository -Name $testRepository -URL $tmpRepoPath -Priority 1 -ErrorAction SilentlyContinue
+        $script:repositoryPath = [IO.Path]::GetFullPath((get-psresourcerepository "testRepository").Url.AbsolutePath)

```

Comment:
This was failing the tests locally due to the difference in expected and actual paths:
`C:\Users\CANSU~1.ERD` vs `C:\Users\cansu.erdogan`